push-build.sh

```diff
@@ -121,6 +123,9 @@ fi
 
 GCS_DEST="devel"
 ((FLAGS_ci)) && GCS_DEST="ci"
+if [[ -n "${FLAGS_release_type:-}" ]]; then
+  GCS_DEST=$FLAGS_release_type
+fi
```

Comment:
This if/then can (but does not need to) be expressed with:

GCS_DEST=${FLAGS_release_type:-$GCS_DEST}